src/login/logind-dbus.c

```diff
@@ -2357,10 +2355,12 @@ static int method_cancel_scheduled_shutdown(sd_bus_message *message, void *userd
                 }
 
                 username = uid_to_name(uid);
-                utmp_wall("The system shutdown has been cancelled",
+                utmp_wall("System shutdown has been cancelled",
```

Comment:
The line below needs to be updated:
https://github.com/systemd/systemd/blob/cd25e170014783503d5035ef4b067e7b00c01221/test/test-shutdown.py#L67

Reply:
I moved that chunk to the other PR.